lib/miq_expression/field.rb

```diff
@@ -82,6 +82,15 @@ def contains(other)
     )
   end
 
+  def column_type
+    if custom_attribute_column?
+      field_type = CustomAttribute.find_by_name(column.gsub(CustomAttributeMixin::CUSTOM_ATTRIBUTES_PREFIX, "")).serialized_value.class
```

Comment:
@alongoldboim 

1) `find_by_name` I guess that this method is obsolete
can you use 
`CustomAttribute.find_by(:name => '' `

2) what if record will be not found ?
If am not sure if it should throw exception or return :string but `type_for_attribute` is returning `nil` in that case.

3) maybe it will be useful to add to method  `custom_attribute_column` (for example)
column.gsub(CustomAttributeMixin::CUSTOM_ATTRIBUTES_PREFIX, "")



what do you think ?